Formula/j/juman.rb

```diff
@@ -27,8 +27,12 @@ class Juman < Formula
   end
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    # Fix compile with newer Clang
+    if DevelopmentTools.clang_build_version >= 1403
+      ENV.append_to_cflags "-Wno-implicit-int -Wno-implicit-function-declaration"
+    end
+
+    system "./configure", *std_configure_args
```

Comment:
```suggestion
    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] }
```

Reply:
It builds fine without adding this. I had updated documentation a while back to just use defaults if they are automatically ignored, similar to how we handle CMake - https://github.com/Homebrew/brew/pull/16408

Reply:
that is true.